packages/docs/components/MethodList/MethodList.tsx

```diff
@@ -37,7 +37,7 @@ export const MethodList: React.FC<MethodListProps> = ({ name, intro, usage, para
         <>
           <MethodBadge label="Parameters" />
 
-          <List bulleted={false}>
+          <List style={{ listStyleType: 'none' }}>
```

Comment:
We still need the `margins/paddings`, just a one-off for the `MethodList` component.